.github/workflows/release-project.yaml

```
@@ -26,11 +26,22 @@ env:
   MAIN_BRANCH: ros2
 
 jobs:
+  variables:
+    runs-on: ubuntu-24.04
+    outputs:
+      RC_BRANCH_NAME: ${{ steps.variables.outputs.RC_BRANCH_NAME }}
+    steps:
+      - name: Set variables
+        id: variables
+        run: |
```

actionlint (1.7.4)

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)